src/screens/SignupScreen/SignupScreen.spec.tsx

```diff
@@ -32,13 +30,10 @@ jest.mock('redux-saga/effects', () => {
     delay: () => () => true,
   };
 });
-
-const someApiResponse = { idToken: 'highly-secure-token', localId: '999abcd' };
+jest.mock('~/utils/firebase');
 
 describe('<SignupScreen />', () => {
-  beforeEach(() => {
-    (signupUser as jest.Mock).mockResolvedValue(someApiResponse);
-  });
+  // TODO: ? should we still use beforeEach with .mockResolvedValue(someApiResponse) here ?
```

Comment:
I can't find to what this comment is related to. Is it needed? Let me know.

Reply:
It is related to the comment https://github.com/k-bialucha/business-assistant-mobile-app/pull/31#discussion_r527137440 and will be removed as well